### PR TITLE
hometask_1

### PR DESCRIPTION
**Pull request by izy-13**

### CODE REVIEW

File: docs/2-building-blocks-of-oop-part-1/hometask/src/Triangle.ts [hunk 1]
```diff
@@ -11,17 +11,15 @@ export class Triangle extends Shape {
     const side1 = this.points[0].distance(this.points[1]);
     const side2 = this.points[1].distance(this.points[2]);
     const side3 = this.points[2].distance(this.points[0]);
-    if (Math.round(side1) === Math.round(side2) && Math.round(side2) === Math.round(side3)) {
+    if (side1.toFixed(1) === side2.toFixed(1) && side2.toFixed(1) === side3.toFixed(1)) {
       return "equilateral triangle";
     } else if (side1 === side2 || side1 === side3 || side2 === side3) {
```
Comment (AlenaBDE) [inline docs/2-building-blocks-of-oop-part-1/hometask/src/Triangle.ts]: remove this else also, just `return`

Comment (izy-13) [reply docs/2-building-blocks-of-oop-part-1/hometask/src/Triangle.ts]: @AlenaBDE did you mean it?

image

Comment (AlenaBDE) [reply docs/2-building-blocks-of-oop-part-1/hometask/src/Triangle.ts]: if mean return after each `if`, no `else` part, 
on the screenshot it is right

Comment (izy-13) [reply docs/2-building-blocks-of-oop-part-1/hometask/src/Triangle.ts]: @AlenaBDE updated

**Pull request merged**